### DISCUSSION TITLE
New Installation, no Login Window

### DISCUSSION THREAD

**Discussion by nicedevil007** [requarks/wiki#6868] [Error / Bug Report]
Hi there,

yesterday I installed my new instance of wiki.js with postgres-16 and node.js 20.
Now I came back (after creating my git repository with gitea) and wanted to set it up in the admin panel.
Unfortunately the page isn't working as expected anymore.

First I get this page:
![image](https://github.com/requarks/wiki/assets/17103076/d3ba06da-4ca6-420a-92f9-4dc07d7f1c62)

After choosing Administration I get this:
![image](https://github.com/requarks/wiki/assets/17103076/467eda53-7386-4ee2-9be7-f7297ce0e5a8)

And then I want to login with my credentials from yesterday:
![image](https://github.com/requarks/wiki/assets/17103076/c87816c1-9863-47d8-8eb1-bc73cb0625a9)

And thats it, no username/password filed anymore. I tried it with IP-Adress directly on the machine, same behaviour. I also tried it with different browsers and in-private windows of different browsers. All the same.

Can someone help me out on starting to debug?

**Comment by NGPixel** [comment]
Check in the browser dev tools if you get an error?

**Reply by nicedevil007** [reply]
Sry forgot to update my post. Already fixed with new installation.

I guess I enabled Force HSTS inside the admin settings. This option is already set in reverseproxy. I guess this was my issue on first installation.
